src/axom/inlet/SphinxWriter.hpp

```diff
@@ -226,13 +207,14 @@ class SphinxDocWriter : public DocWriter
   */
   std::string getValidStringValues(axom::sidre::Group* sidreGroup);
 
-  axom::sidre::Group* m_sidreRootGroup;
   std::ofstream m_outFile;
   std::ostringstream m_oss;
   // This is needed to preserve the traversal order of the Inlet::Tables
   std::vector<std::string> m_inletTablePathNames;
   std::unordered_map<std::string, TableData> m_rstTables;
   std::string m_fileName;
+  // Used for the RST tables for fields
+  std::vector<std::string> m_colLabels;

```

Comment:
Minor: Since you already depend on Sidre, you might consider using a `sidre::MapCollection` for this.
The data is stored in an integer-indexed array, but entries can be indexed by strings.

See: https://github.com/LLNL/axom/blob/develop/src/axom/sidre/core/MapCollection.hpp

Reply:
I agree that something like this might be useful, but the use of raw pointers as the stored item instead of objects themselves would require introducing manual memory management here (I think) since I want the data to be owned by the container.

I'll have to tweak this logic a bit anyways when I add callback function documentation, so possibly a more elegant solution will present itself.